pkgs/by-name/dp/dprint/plugins/rubixdev-stylua.nix

```diff
@@ -0,0 +1,22 @@
+{ pkgs, mkDprintPlugin, ... }:
+let
+  version = "v0.2.1";
+  homepage = "https://github.com/RubixDev/dprint-plugin-stylua";
+  changelog = "${homepage}/releases/${version}";
+in
+mkDprintPlugin {
+  description = "Format Lua code through dprint using StyLua";
+  hash = "sha256-atCb2iS7TJhiX3w5ngf9gmMe3RgghR3MSykwHPxbYLk=";
+  initConfig = {
+    configExcludes = [ ];
+    configKey = "stylua";
+    fileExtensions = [
+      "lua"
+    ];
+  };
+  pname = "rubixdev-stylua";
+  updateUrl = "https://plugins.dprint.dev/RubixDev/stylua/latest.json";
+  url = "https://plugins.dprint.dev/RubixDev/stylua-${version}.wasm";
+  license = pkgs.lib.licenses.gpl3Plus;
+  inherit version homepage changelog;
```

Comment:
these three can be made to be hardcoded like the official plugins. but `license` and `maintainers` cannot be.